packages/frontend/src/components/NavBar/ExploreMenu/index.tsx

```diff
@@ -175,6 +174,7 @@ const ExploreMenu: FC<Props> = memo(({ projectUuid }) => {
             )}
 
             <DashboardCreateModal
+                projectUuid={projectUuid}
```

Comment:
good catch. I think `useCreateMutation` and rest of the effects and redirects are not necessary anymore here

Reply:
I removed the mutate itself, but I think we still need the loading and completed for redirection